src/Sarif.UnitTests/Writers/PrereleaseCompatibilityTransformerTests.cs

```diff
@@ -22,7 +41,31 @@ public void PrereleaseCompatibilityTransformer_UpgradesPrereleaseTwoZeroZero()
             PrereleaseCompatibilityTransformer.UpdateToCurrentVersion(sarifText, forceUpdate: false, formatting: Formatting.None, out sarifText);
 
             SarifLog sarifLog = JsonConvert.DeserializeObject<SarifLog>(sarifText);
-            JsonConvert.SerializeObject(sarifLog);            
+            JsonConvert.SerializeObject(sarifLog);
+        }
+
+        [Fact]
+        public void PrereleaseCompatibilityTransformer_NestedFiles()
```

Comment:
>PrereleaseCompatibilityTransformer_NestedFiles [](start = 20, length = 46)

these tests moved from a deleted file. I put a separate prerelease transformer tests file in the wrong location (visitor dir rather than writers)